cookbook/multi-llm-discussion.py

```
@@ -0,0 +1,284 @@
+
+        Returns
+        -------
+        str
+            The history as seen by the member
+        """
+        relevant_messages: list[DiscussionMessage] = []
+        for message in self.conversation:
+            if isinstance(message, BotMessage):
+                relevant_messages.append(message)
+            elif isinstance(message, ModeratorMessage) and member in message.visible_to:
+                relevant_messages.append(message)
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)